src/main/java/nl/novi/techiteasy/models/Television.java

```diff
@@ -0,0 +1,216 @@
+package nl.novi.techiteasy.models;
+
+import javax.persistence.*;
+
+@Entity
+public class Television {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "type")
```

Comment:
De default-name van je column is de naam van je variabele. De naam die je hier invult in je @Comlumn annotatie, is hetzelfde als je variabele naam.